secboot/secboot_hooks.go

```diff
@@ -75,18 +75,10 @@ func SealKeysWithFDESetupHook(runHook fde.RunSetupHookFunc, keys []SealKeyReques
 }
 
 func writeKeyData(path string, keySetup *fde.InitialSetupResult, auxKey []byte, model sb.SnapModel) error {
-	var handle []byte
-	if keySetup.Handle == nil {
-		// this will reach fde-reveal-key as null but should be ok
-		handle = []byte("null")
-	} else {
-		handle = *keySetup.Handle
-	}
```

Comment:
why we don't need this bit of code/manipulation anymore?

Reply:
I have no idea. I believe that was done by Michael. I suppose we can revert this one if it breaks customers' `fde-reveal-key`.

Reply:
I suppose, if "fde-setup" does not set a handle. It probably does not care about the handle in the corresponding "fde-reveal-key".